client-app/pages/account/edit-quote.vue

```diff
@@ -209,6 +200,22 @@ const quoteValid = computed<boolean>(
 
 const userHasAddresses = computedEager<boolean>(() => !!accountAddresses.value.length);
 
+const isQuoteCanBeSaved = computed<boolean>(
```

Comment:
I think you can reuse `quoteValid` here to simplify this overcomplicated logic

Reply:
@asvishnyakov we can not reuse quoteValid, because we have to detect changes and we can save even invalid quote.